source/mainnet/docs/guides/send-ccd-wallets.rst

```diff
@@ -20,35 +20,30 @@ This topic describes how you can send CCD from an account when only one particip
 
 .. dropdown:: |cryptox|
 
-    #. On the Accounts screen tap |cryptoX-send| on the account from which you want to send CCD. You can also tap |cryptoX-send| on the account's balance screen.
+    #. Tap **Send** on the main screen.
 
-    #. Enter the amount you want to transfer.
+    #. Enter the amount you want to transfer and specify the recipient. You can paste the recipient address, choose a recipient that already exists in your address book, or tap the Scan QR icon to add a new one.
```

Comment:
Scanning QR doesn't add a new recipient to the address book. It just lets the scanned account to be the recipient of this transfer